.github/workflows/test.yml

```diff
@@ -88,22 +87,18 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.7
+          python-version: 3.8
       - name: Install Mitiq
         run: |
           python -m pip install --upgrade pip
           pip install .
 
       - name: Run ASV benchmarks
         run: |
-          pip install asv
+          pip install asv virtualenv
+          export BENCHMARK_CI=1
           asv machine --yes
-          result=$(asv dev)
-          echo "${result}"
-          if [[ $result == *"failed"* ]]; then
-            exit 1
-          fi
-          exit 0
+          asv run --quick --strict
```

Comment:
Does this exit 0 if there is an error?

Reply:
It will return a non-zero error code when there is an error. From the [`asv run`](https://asv.readthedocs.io/en/stable/commands.html?highlight=--quick#asv-run) docs:

```
 --strict              When set true the run command will exit with a non-
                       zero return code if any benchmark is in a failed state
```